### PR TITLE
Added Debian Install Fix + WSL2 Instructions

### PR DESCRIPTION
**Pull request by diveyez**
Added more **README.md** documentation for setting up with Debian, Raspberry Pi's and WSL.

Added informational links with information on how to setup **Visual Studio Code** to connect to **Docker** through **WSL2** for allowing users to directly interact with **SuperAlgos** through **VSCode** with ease and freedom from requiring anything more than an IDE to operate and start contributing to **SuperAlgos**.

**Edited File:**
[README.md](https://github.com/Superalgos/Superalgos/files/7561753/README.md)

_**Discovery,
Using VSCode with WSL/2 leads to new abilities regarding the stability while live deploying and editing of code on Windows.
WSL2+ now allows containers to become VM's and vice versa.
I will be looking into providing a CI pipeline that builds a working WSL VM which automatically installs a remotely accessible interface as SuperAlgos ages.**_

### CODE REVIEW

**Review by diveyez — COMMENTED** [review]
As it should be. Just needed to clarify a few more things.

File: README.md [hunk 1]
```diff
@@ -212,14 +212,15 @@ nvm run node <command string/var>
 __There is a few parts that are necessary to obtain full functionality from VSCode as an app to run notebooks for ML/AI algos and you can turn VSCode and Windows into a development bench for working with Super Algos.__
 
 - First you need to install WSL and WSL2 [https://docs.microsoft.com/en-us/windows/wsl/install](https://docs.microsoft.com/en-us/windows/wsl/install) then reboot if promoted.
-  - You may want to review the Docker WSL2 Backend information as well [https://aka.ms/vscode-remote/containers/docker-wsl2](https://aka.ms/vscode-remote/containers/docker-wsl2)
+  - You may want to review the Docker WSL2 Backend information for VSCode as well before proceeding. [https://aka.ms/vscode-remote/containers/docker-wsl2](https://aka.ms/vscode-remote/containers/docker-wsl2)
```
Comment (diveyez) [inline README.md:215]: Keep

**Comment by Luis-Fernando-Molina** [comment]
hey! thanks this is great. 

Are you already in contact with @NAmorim ? I believe you 2 should team up.

**Comment by diveyez** [comment]
> 
> 
> hey! thanks this is great.
> 
> Are you already in contact with @NAmorim ? I believe you 2 should team up.


That sounds great I will do that and make this pull request on the development branch.

**Pull request closed without merging**